groundstation/static/js/App.jsx

```diff
@@ -53,8 +53,20 @@ function App() {
   }
   const classes = useStyles();
   const [open, setOpen] = React.useState(false);
+  const [isAdmin, setIsAdmin] = React.useState(false);
   const anchorRef = React.useRef(null);
 
+  const auth_token = sessionStorage.getItem('auth_token');
+    fetch(`/api/users/${auth_token}`).then(results => {
+        return results.json();
+    }).then(data => {
+        if (data.status === 'success') {
+            setIsAdmin(data.data.is_admin);
+        } else {
+            console.error('Unexpected error occured:');
+        }
+    });
```

Comment:
Can you move this block of code (LInes 59-68) below line 52 (inside the `if (isAuthenticated()) { ... }` block)? It prevents an error from being raised when the front page is loaded without a login (when auth_token doesn't exist in session storage)